.github/workflows/benchmark-pr.yaml

```diff
@@ -0,0 +1,26 @@
+name: benchmark
+
+on: pull_request
```

Comment:
We can use `path-ignore` to ignore only docs/mixins change
```
on:
  push:
    paths-ignore:
    - 'docs/**'
```

Reply:
Actually wow! We can divide those things in current CI.. just we need to move to GH actions! That's amazing.

I think we should create an GH issuse for that (:

Reply:
https://github.com/thanos-io/thanos/issues/2257